syntax = "proto3";

package rpc_package;

enum ErrorCode {
    OK             = 0;
    FAIL           = -1;
}
